onoff.js

```diff
@@ -240,6 +240,18 @@ class Gpio {
       // the bbb.
     }
   }
+
+  static get accessible() {
+    try {
+      fs.openSync(GPIO_ROOT_PATH + 'export', 'r');
+    } catch(e) {
+      if(e.code === 'EACCESS') {
```

Comment:
I have a few questions here

 * Which operating systems support 'EACCESS'?
 * Is there a typo here and should 'EACCESS' be 'EACCES'?
 * On which operating systems was the PR tested?
 * Which operating systems does the PR support?
 * What happens if the code is run with `sudo` privileges?
 * Why is 'EACCESS' used for the test?

Edit:
 * What about other errors, for example, ENOENT?